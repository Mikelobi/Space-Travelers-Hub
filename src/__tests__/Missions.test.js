import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Missions from '../components/Pages/Mission';

const mission = [
  {
    desc: 'Eutelsat S.A. is a European satellite operator',
    key: 'special',
    id: 1,
    name: 'defines the satellite',
    joining: true,
  },
];

describe('testing the rendering of missions', () => {
  it('', () => {
    const { container } = render(
      <Provider store={store}>
        <Missions key={mission[0]} mission={mission[0]} />
      </Provider>,
    );
    expect(container).toMatchSnapshot();
  });
});
