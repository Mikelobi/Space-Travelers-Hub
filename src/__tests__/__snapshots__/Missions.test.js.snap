// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the rendering of missions  1`] = `
<div>
  <section
    class="missionsContainer"
  >
    <h2
      class="header"
    >
      Missions Page
    </h2>
    <table
      class="table table-striped table-bordered table-hover"
    >
      <thead>
        <tr>
          <th>
            Mission Name
          </th>
          <th>
            Mission Description
          </th>
          <th>
            Mission Status
          </th>
          <th>
            Actions
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </section>
</div>
`;
